include/TrackContentWidget.h

```diff
@@ -82,17 +88,37 @@ class TrackContentWidget : public QWidget, public JournallingObject
 
 	QBrush darkerColor() const;
 	QBrush lighterColor() const;
-	QBrush gridColor() const;
+	QBrush coarseGridColor() const;
+	QBrush fineGridColor() const;
+	QBrush horizontalColor() const;
 	QBrush embossColor() const;
 
-	void setDarkerColor( const QBrush & c );
-	void setLighterColor( const QBrush & c );
-	void setGridColor( const QBrush & c );
-	void setEmbossColor( const QBrush & c);
+	int coarseGridWidth() const;
+	int fineGridWidth() const;
+	int horizontalWidth() const;
+	int embossWidth() const;
+
+	int embossOffset() const;
+
+	void setDarkerColor(const QBrush & c);
+	void setLighterColor(const QBrush & c);
+	void setCoarseGridColor(const QBrush & c);
+	void setFineGridColor(const QBrush & c);
+	void setHorizontalColor(const QBrush & c);
+	void setEmbossColor(const QBrush & c);
+
+	void setCoarseGridWidth(const int & c);
+	void setFineGridWidth(const int & c);
+	void setHorizontalWidth(const int & c);
+	void setEmbossWidth(const int & c);
+
+	void setEmbossOffset(const int& c);
```

Comment:
There's no need to take `int`s by const reference - passing by value makes more sense.